### PR TITLE
added new question in Q45 with answer

### PR DESCRIPTION
**Pull request by Jian-Chueng**
Dear @Ebazhanov  and @Hephaest,

I added one new question for Matlab with answer in Q45.

Please review and add it.

Thanks for sharing,

Jian

### CODE REVIEW

File: matlab/matlab-quiz.md [hunk 1]
```diff
@@ -758,3 +758,16 @@ b = "stand"
 - [ ] ischar(b)
 - [ ] length(a) == length(b)
 - [ ] class(a) == class(b)
+
+#### Q45. Which does E contain?
+```MATLAB
```
Comment (Hephaest) [inline matlab/matlab-quiz.md:763]: `markdownlint` issue occurs:

> MD031/blanks-around-fences: Fenced code blocks should be surrounded.

It is better to check the `markdownlint` before pulling a request next time. 😄

**Review by Hephaest — APPROVED** [review]
Not a big issue. Well Done!

**Review by Ebazhanov — APPROVED** [review]
Thank you @Jian-forks, and sorry for the delay with Approve

**Comment by Ebazhanov** [comment]
@all-contributors please add @Jian-forks for design, code

**Pull request merged**